revista-medica-del-instituto-mexicano-del-seguro-social.csl

```diff
@@ -0,0 +1,445 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="es-MX">
+  <info>
+    <title>Revista Médica del Instituto Mexicano del Seguro Social (Español)</title>
+    <title-short>IMSS</title-short>
+    <id>http://www.zotero.org/styles/revista-medica-del-instituto-mexicano-del-seguro-social</id>
+    <link href="http://www.zotero.org/styles/revista-medica-del-instituto-mexicano-del-seguro-social" rel="self"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="https://revistamedica.imss.gob.mx/index.php/revista_medica/libraryFiles/downloadPublic/31" rel="documentation"/>
+    <author>
+      <name>Omar Chávez Martínez</name>
+      <email>omar.chavezm@imss.gob.mx</email>
+      <uri>https://orcid.org/0000-0003-2633-1898</uri>
+    </author>
+    <contributor>
+      <name>Alejandro Moctezuma Paz</name>
+      <email>alejandro.moctezuma@imss.gob.mx</email>
+      <uri>https://orcid.org/0000-0003-4015-8040</uri>
+    </contributor>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <eissn>2448-5667</eissn>
+    <issn>0443-5117</issn>
+    <summary>Adaptación del Estilo Vancouver para manuscritos que serán enviados a la Revista Médica del Instituto Mexicano del Seguro Social</summary>
+    <updated>2025-10-22T00:00:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <!-- Localización español -->
+  <locale xml:lang="es">
+    <date form="text" delimiter=" ">
+      <date-part name="day"/>
+      <date-part name="month" form="short" strip-periods="true"/>
+      <date-part name="year"/>
+    </date>
+    <terms>
+      <term name="collection-editor" form="long">
+        <single>editor</single>
+        <multiple>editores</multiple>
+      </term>
+      <term name="editor" form="long">
+        <single>editor</single>
+        <multiple>editores</multiple>
+      </term>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+      <term name="volume" form="short">vol.</term>
```

Comment:
As mentioned, please go through all your locale overrides and removed duplications or just minor stylings like capitalisation of terms already included in the locale. I still spot at least 2...
https://github.com/citation-style-language/locales/blob/master/locales-es-MX.xml